src/vector/geometry.rs

```diff
@@ -43,7 +43,7 @@ impl Geometry {
         *(self.c_geometry_ref.borrow_mut()) = Some(c_geometry);
     }
 
-    unsafe fn with_c_geometry(c_geom: OGRGeometryH, owned: bool) -> Geometry {
+    pub unsafe fn with_c_geometry(c_geom: OGRGeometryH, owned: bool) -> Geometry {
```

Comment:
I'm not sure if we want to pub this method. Could you change it to pub(crate) ?

Reply:
Updated.